### PR TITLE
Corrects typo & general codestyle improvements

### PR DESCRIPTION
**Pull request by nikolajlovenhardt**

### CODE REVIEW

File: src/Twilio.php [hunk 1]
```diff
@@ -119,12 +122,12 @@ private function twiml(callable $callback)
...
         $message = new Services_Twilio_Twiml();
 
-        if (is_callable($callback)) {
-            call_user_func($callback, $message);
-        } else {
+        if (!is_callable($callback)) {
             throw new InvalidArgumentException('Callback is not valid.');
         }
 
+        call_user_func($callback, $message);
```
Comment (hannesvdvreken) [inline src/Twilio.php]: early return :+1:

File: src/Support/Laravel/L5ServiceProvider.php [hunk 2]
```diff
@@ -15,10 +15,10 @@ class L5ServiceProvider extends LaravelServiceProvider
     public function boot()
     {
         $this->publishes([
-            __DIR__.'/../../config/config.php' => config_path('twilio.php'),
+            __DIR__ . '/../../config/config.php' => config_path('twilio.php'),
         ]);
 
-        $this->mergeConfigFrom(__DIR__.'/../../config/config.php', 'twilio');
+        $this->mergeConfigFrom(__DIR__ . '/../../config/config.php', 'twilio');
```
Comment (hannesvdvreken) [inline src/Support/Laravel/L5ServiceProvider.php]: symfony style guide is a bit stricter than psr-2. This includes removing all spaces surrounding `.` (concatenation character)

File: src/Support/Laravel/ServiceProviderTrait.php [hunk 3]
```diff
@@ -18,16 +18,16 @@ public function register()
...
-        $this->app->singleton('twilio.sms', 'Aloha\Twilio\Commands\TwilioSmsCommand');
+        $this->app->singleton('twilio.sms', \Aloha\Twilio\Commands\TwilioSmsCommand::class);
 
         // Register Twilio Test Call Command.
-        $this->app->singleton('twilio.call', 'Aloha\Twilio\Commands\TwilioCallCommand');
+        $this->app->singleton('twilio.call', \Aloha\Twilio\Commands\TwilioCallCommand::class);
 
         // Register TwilioInterface concretion.
-        $this->app->singleton('Aloha\Twilio\TwilioInterface', function () {
+        $this->app->singleton(\Aloha\Twilio\TwilioInterface::class, function () {
```
Comment (hannesvdvreken) [inline src/Support/Laravel/ServiceProviderTrait.php]: please import these

File: src/Twilio.php [hunk 4]
```diff
@@ -84,8 +84,11 @@ public function call($to, $message, array $options = [], $from = null)
         $twilio = $this->getTwilio();
 
         if (is_callable($message)) {
-            $query = http_build_query(['Twiml' => $this->twiml($message)]);
-            $message = 'https://twimlets.com/echo?'.$query;
+            $query = http_build_query([
+                'Twiml' => $this->twiml($message)
```
Comment (hannesvdvreken) [inline src/Twilio.php]: trailing comma

File: src/Twilio.php [hunk 5]
```diff
@@ -84,8 +84,11 @@ public function call($to, $message, array $options = [], $from = null)
...
+            ]);
+
+            $message = 'https://twimlets.com/echo?' . $query;
```
Comment (hannesvdvreken) [inline src/Twilio.php]: please remove spaces surrounding the `.`

**Comment by hannesvdvreken** [comment]
I'd love to merge this

File: src/Twilio.php (hunk 1 above)
Comment (nikolajlovenhardt) [inline src/Twilio.php]: @hannesvdvreken Do you want me to change anything here?

Comment (hannesvdvreken) [inline src/Twilio.php]: nope

**Comment by hannesvdvreken** [comment]
thank you

**Pull request merged**